server/router.js

```diff
@@ -233,6 +233,38 @@ router.get('/api/group/:groupId/articles/paginated', (req, res) => {
   });
 });
 
+router.get('/api/public/articles/paginated', (req, res) => {
+  const conditions = { pagination: {}, sort: {} };
+
+  // defaults
+  let limit = Number.parseInt(req.query.limit, 10);
```

Comment:
Will it still return articles if I don't give it any pagination information in the query?